x-pack/plugins/alerting/server/application/rule/methods/bulk_delete/bulk_delete_rules.ts

```diff
@@ -71,20 +90,35 @@ export const bulkDeleteRules = async (context: RulesClientContext, options: Bulk
   ]);
 
   const deletedRules = rules.map(({ id, attributes, references }) => {
-    return getAlertFromRaw(
-      context,
+    // TODO (http-versioning): alertTypeId should never be null, but we need to
+    // fix the type cast from SavedObjectsBulkUpdateObject to SavedObjectsBulkUpdateObject
+    // when we are doing the bulk create and this should fix itself
+    const ruleType = context.ruleTypeRegistry.get(attributes.alertTypeId!);
+    const ruleDomain = transformRuleAttributesToRuleDomain<Params>(attributes as RuleAttributes, {
```

Comment:
nitpick: The comments say bulk create, but i think it's because of `retryIfBulkOperationConflicts`'s return type right? Let's change the comment to reflect that.